### DISCUSSION TITLE
[Vote ended on 2022-11-04] Change release cadence and schedule of Ansible from 7.0.0 on to couple more tightly to ansible-core

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#157] [Votes done]
Discussion: https://github.com/ansible-community/community-topics/issues/151
Concrete proposal to vote on: https://github.com/ansible-community/community-topics/issues/151#issuecomment-1293919082

> 1. Release 7.0.0rc1 one week after 7.0.0b1, where 7.0.0b1 is released approx. one day after ansible-core 2.14.0.
> 2. If there is no major problem with 7.0.0rc1 reported until the end of the week when it was released, the 7.0.0 release will happen one week after the 7.0.0rc1 release.
> 3. Otherwise 7.0.0rc2 wlil be released one week after the 7.0.0rc1 release, and 7.0.0 one week after 7.0.0rc2.
> 4. The 7.X.0 release will happen appox. one day after 2.14.X for X > 0.

This would be the expected schedule:

- 2022-11-07 ansible-core 2.14.0
- 2022-11-08/09 Ansible 7.0.0b1
- 2022-11-15 Ansible 7.0.0rc1
- 2022-11-18 major defects in Ansible 7.0.0rc1 must be know by then to trigger a rc2 release
- 2022-11-22 Ansible 7.0.0 or Ansible 7.0.0rc2 (when necessary)
- 2022-11-29 (Ansible 7.0.0 when rc2 was necessary)
- 2022-12-05 ansible-core 2.14.1
- 2022-12-06 Ansible 7.1.0

Please vote +1 to accept the proposal (with the new schedule), or -1 to reject it. When rejected, we will stay with the current schedule as in https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/roadmap/COLLECTIONS_7.rst#release-schedule.

**Comment by felixfontein** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by gundalow** [comment]
+1
